docs/standard/microservices-architecture/architect-microservice-container-applications/scalable-available-multi-container-microservice-applications.md

```diff
@@ -13,31 +13,41 @@ ms.topic: article
 
-Figure 4-22 illustrates deployment into a cluster of an application composed of multiple microservices (containers).
+Figure 4-23 illustrates deployment into a cluster of an application composed of multiple microservices (containers).
 
-![](./media/image22.PNG)
+![](./media/image23.PNG)
 
-**Figure 4-22**. A cluster of containers
+**Figure 4-23**. A cluster of containers
 
 It looks like a logical approach. But how are you handling load-balancing, routing, and orchestrating these composed applications?
 
-The Docker CLI meets the needs of managing one container on one host, but it falls short when it comes to managing multiple containers deployed on multiple hosts for more complex distributed applications. In most cases, you need a management platform that will automatically start containers, suspend them or shut them down when needed, and ideally also control how they access resources like the network and data storage.
+The plain Docker Engine in single Docker hosts meets the needs of managing single image instances on one host, but it falls short when it comes to managing multiple containers deployed on multiple hosts for more complex distributed applications. In most cases, you need a management platform that will automatically start containers, sacale-out containers with multiple instances per image, suspend them or shut them down when needed, and ideally also control how they access resources like the network and data storage.
 
 To go beyond the management of individual containers or very simple composed apps and move toward larger enterprise applications with microservices, you must turn to orchestration and clustering platforms.
 
 From an architecture and development point of view, if you are building large enterprise composed of microservices-based applications, it is important to understand the following platforms and products that support advanced scenarios:
 
-**Clusters and orchestrators**. When you need to scale out applications across many Docker hosts, as when a large microservice-based application, it is critical to be able to manage all those hosts as a single cluster by abstracting the complexity of the underlying platform. That is what the container clusters and orchestrators provide. Examples of orchestrators are Docker Swarm, Mesosphere DC/OS, Kubernetes (the first three available through Azure Container Service) and Azure Service Fabric.
+**Clusters and orchestrators**. When you need to scale out applications across many Docker hosts, as when a large microservice-based application, it is critical to be able to manage all those hosts as a single cluster by abstracting the complexity of the underlying platform. That is what the container clusters and orchestrators provide. Examples of orchestrators are Azure Service Fabric, Kubernetes, Docker Swarm and Mesosphere DC/OS,  (the last three available through Azure Container Service).
```

Comment:
,  (the last three available through Azure Container Service). -> These orchestrators are available through Azure Container Service.

Reply:
But, Service Fabric is NOT available through ACS... I'm changing the way, though.

Reply:
Sorry, I misread and only saw 3 in the list.